service/src/main/java/uk/gov/hmcts/reform/fpl/testingsupport/controllers/TestingSupportController.java

```diff
@@ -80,4 +91,29 @@ public void populateCase(@PathVariable("caseId") Long caseId, @RequestBody Map<S
             throw e;
         }
     }
+
+    @PostMapping(value = "/testing-support/case/{caseId}/access")
+    public void grantCaseAccess(@PathVariable("caseId") Long caseId, @RequestBody Map<String, String> requestBody) {
```

Comment:
same as comment on e2e, maybe can exist in separate PR if not relevant to respondent statement?

Reply:
same as above, this is used by added e2e test